app/api/auth.py

```diff
@@ -116,6 +117,9 @@ def log_request(request, key):
     method = request.method
     path = request.path
     user = key.email
-    payload = request.json
+    try:
+        payload = request.get_json()
+    except HTTPException:
+        payload = None
```

Comment:
Looking at the [documentation for `request.get_json()`](https://flask.palletsprojects.com/en/1.1.x/api/#flask.Request.get_json), I would think that we can go ahead and just pass `silent=True` to it rather than do this whole try/except thing with an additional import.

```python
    user = key.email
    payload = request.get_json(silent=True)
    logger = create_logger if method == "POST" else update_logger
```

This works for me locally

Reply:
Ah okay. this is definitely cleaner. I'll just change it as you suggested.